stix_shifter_modules/vectra/stix_translation/json/to_stix_map.json

```diff
@@ -52,18 +52,6 @@
       "key": "x-ibm-finding.description",
       "object": "detection"
     },
-    "num_sessions": {
-      "key": "x-ibm-finding.event_count",
-      "object": "detection"
-    },
-    "num_attempts": {
-      "key": "x-ibm-finding.event_count",
-      "object": "detection"
-    },
-    "num_successes": {
-      "key": "x-ibm-finding.x_num_successes",
-      "object": "detection"
-    },
```

Comment:
is a transformer not working for these field values?

Reply:
Transformer isn't applicable as there is no conversion required. 
This attribute is specific to a particular detection type from data source only.
If query results in number of detections where at least one detection consists of this attribute then it gets added to those detection types where it is not present, with value 'NaN' in STIX output.
And this happens from CP4S side. In CLI,  this attribute is present in those particular stix records only.